templates/terraform/examples/billing_budget_basic.tf.erb

```diff
@@ -0,0 +1,18 @@
+data "google_billing_account" "account" {
+  billing_account = "<%= ctx[:test_env_vars]['billing_acct'] -%>"
+}
+
+resource "google_billing_budget" "<%= ctx[:primary_resource_id] %>" {
+  provider = google-beta
+  billing_account = data.google_billing_account.account.id
+  display_name = "<%= ctx[:vars]['display_name'] %>"
+  amount {
+    specified_amount {
+      currency_code = "USD"
+      units = "10"
```

Comment:
```suggestion
      units = "100000"
```

since we'll be running these tests in a real environment, let's set this number high so we don't accidentally start seeing alerts

Reply:
Done.